test/cli/test_cli_functions.sh

```diff
@@ -26,7 +26,13 @@ function kuberun {
 function noobaa {
     local rc
     local run_with_timeout=false
+    local should_fail=false
     local silence=false
+    if [ "${1}" == "failure" ]
```

Comment:
I think this is becoming very confusing that we have an override of the noobaa command that has all these test options that do not exist in the CLI. At least I would rename it to test_noobaa or something.

Reply:
changed to test_noobaa